<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FirebirdProcedureForm</class>
 <widget class="QDialog" name="FirebirdProcedureForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>895</width>
    <height>629</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tGeneral">
      <attribute name="title">
       <string>General</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QLabel" name="lblName">
         <property name="text">
          <string>Name</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="edtName"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QLabel" name="lblInputArguments">
             <property name="text">
              <string>Procedure arguments</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QTableView" name="tvInputArguments">
             <attribute name="horizontalHeaderStretchLastSection">
              <bool>true</bool>
             </attribute>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QPushButton" name="btnAddInputArg">
               <property name="text">
                <string>Add</string>
               </property>
               <property name="icon">
                <iconset resource="../../sdk/icons.qrc">
                 <normaloff>:/icons/table_row_add.png</normaloff>:/icons/table_row_add.png</iconset>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btnDeleteInputArg">
               <property name="text">
                <string>Delete</string>
               </property>
               <property name="icon">
                <iconset resource="../../sdk/icons.qrc">
                 <normaloff>:/icons/table_row_delete.png</normaloff>:/icons/table_row_delete.png</iconset>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_3">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QLabel" name="lblOutputArguments">
             <property name="text">
              <string>Return arguments</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QTableView" name="tvOutputArguments">
             <attribute name="horizontalHeaderStretchLastSection">
              <bool>true</bool>
             </attribute>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QPushButton" name="btnAddOutputArg">
               <property name="text">
                <string>Add</string>
               </property>
               <property name="icon">
                <iconset resource="../../sdk/icons.qrc">
                 <normaloff>:/icons/table_row_add.png</normaloff>:/icons/table_row_add.png</iconset>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btnDeleteOutputArg">
               <property name="text">
                <string>Delete</string>
               </property>
               <property name="icon">
                <iconset resource="../../sdk/icons.qrc">
                 <normaloff>:/icons/table_row_delete.png</normaloff>:/icons/table_row_delete.png</iconset>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_2">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tSource">
      <attribute name="title">
       <string>Source</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QPlainTextEdit" name="teSource"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnClose">
       <property name="text">
        <string>Close</string>
       </property>
       <property name="icon">
        <iconset resource="../../sdk/icons.qrc">
         <normaloff>:/icons/cross.png</normaloff>:/icons/cross.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnApply">
       <property name="text">
        <string>Apply</string>
       </property>
       <property name="icon">
        <iconset resource="../../sdk/icons.qrc">
         <normaloff>:/icons/tick.png</normaloff>:/icons/tick.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../sdk/icons.qrc"/>
  <include location="../../sdk/icons.qrc"/>
  <include location="../../sdk/icons.qrc"/>
  <include location="../../sdk/icons.qrc"/>
  <include location="../../sdk/icons.qrc"/>
  <include location="../../sdk/icons.qrc"/>
 </resources>
 <connections/>
</ui>
